airbase/airtable.py

```diff
@@ -193,13 +217,7 @@ async def get_bases(self) -> Optional[List[Base]]:
                 self.logger.info(f"Fetched: {len(self.bases)} bases")
 
             else:
-                error_msg = self.get_error_message(
-                    method="get",
```

Comment:
do you care about losing any of this info (method / obj)?

Reply:
The updated `get_error_message()` coroutine leverages the response to determine the method and object of the request